frontend/src/__tests__/cypress/cypress/pages/pipelines/pipelinesTable.ts

```diff
@@ -31,9 +32,119 @@ class PipelineVersionsTableRow extends TableRow {
   }
 }
 
+export enum PipelineSort {
+  PipelineAsc = 'PIPELINE_ASC',
+  PipelineDesc = 'PIPELINE_DESC',
+  Pipeline = 'PIPELINE_SORT',
+  CreatedAsc = 'CREATED_ASC',
+  CreatedDesc = 'CREATED_DESC',
+  Created = 'CREATED_SORT',
+  All = 'SORT_ALL',
+}
+
 class PipelinesTable {
   private testId = 'pipelines-table';
 
+  shouldSortTable({
+    sortType,
+    pipelines,
+    projectName = 'test-project-name',
+  }: {
+    sortType: PipelineSort;
+    pipelines: PipelineKFv2[];
+    projectName?: string;
+  }): void {
+    const sortPipelineByAscending = (): void => {
+      cy.interceptOdh(
+        'GET /api/service/pipelines/:namespace/:serviceName/apis/v2beta1/pipelines',
+        {
+          path: { namespace: projectName, serviceName: 'dspa' },
+          query: { sort_by: 'name asc' },
+        },
+        buildMockPipelines(pipelines),
+      ).as('asc');
+      this.findTableHeaderButton('Pipeline').click();
+      this.findTableHeaderButton('Pipeline').should(be.sortAscending);
+      cy.wait('@asc').should('exist');
```

Comment:
Should be sufficient to simply wait without the `should`
```suggestion
      cy.wait('@asc');
```